beacon-chain/p2p/peers/status.go

```diff
@@ -0,0 +1,311 @@
+// Package peers provides information about peers at the Ethereum protocol level.
```

Comment:
This is a very welcome change. Thanks so much for adding this!

Reply:
You are very welcome.